src/torchmetrics/nominal/cramers.py

```diff
@@ -59,9 +60,9 @@ class CramersV(Metric):
         tensor(0.5284)
     """
 
-    full_state_update = False
-    is_differentiable = False
-    higher_is_better = False
+    full_state_update: bool = False
+    is_differentiable: bool = False
+    higher_is_better: bool = False
```

Comment:
If `1 meaning the perfect association` then this should be true
```suggestion
    higher_is_better: bool = True
```